src/rgw/rgw_d3n_cacherequest.h

```diff
@@ -79,31 +90,32 @@ struct D3nL1CacheRequest : public D3nCacheRequest {
       free(io_buf);
       return -1;
```

Comment:
same as above, why do we propagate bare integer result codes?

Reply:
right, changed to `-EIO`